xbmc/music/MusicDatabase.cpp

```diff
@@ -2945,11 +2945,11 @@ void CMusicDatabase::Clean()
   // other writing access to the database is prohibited.
   if (g_application.IsMusicScanning())
   {
-    CGUIDialogOK::ShowAndGetInput(189, 14057, 0, 0);
+    CGUIDialogOK::ShowAndGetInput(189, 14057);
     return;
   }
 
-  if (CGUIDialogYesNo::ShowAndGetInput(313, 333, 0, 0))
+  if (CGUIDialogYesNo::ShowAndGetInput(313, 333))
```

Comment:
@ronie, sure, but PR is for OK only and YesNo requires way more parameters, so it's wrong either way :)